parlai/core/fixed_data_teacher.py

```diff
@@ -87,6 +161,30 @@ class FixedDataTeacher(Teacher):
 
 
     """
+    @staticmethod
+    def add_cmdline_args(argparser):
```

Comment:
can we only add these if bs>1 ?  hmm, doesnt look easy does it?

Reply:
we can pop then from the opt dict in params.py if batchsize is not greater than one, so they will never be seen by the user.

Reply:
oh sweet!